ark/utils/spatial_analysis_utils.py

```diff
@@ -124,8 +149,12 @@ def compute_close_cell_num(dist_mat, dist_lim, num, analysis_type,
     mark1_num = []
     mark1poslabels = []
 
+    # dist_mat_bin = np.zeros(dist_mat.shape, dtype='int')
     dist_mat_bin = np.zeros(dist_mat.shape, dtype='int')
-    dist_mat_bin[dist_mat < dist_lim] = 1
+    dist_mat_bin[dist_mat.values < dist_lim] = 1
+
+    dist_mat_bin = xr.DataArray(dist_mat_bin,
+                                coords=[dist_mat.dim_0.values, dist_mat.dim_1.values])
```

Comment:
can't we have `coords = dist_mat.coords`?

Reply:
👍